bool hasClockStarted
int32 durationSinceStart
